src/test/java/io/confluent/examples/streams/SessionWindowsExampleTest.java

```diff
@@ -48,57 +48,50 @@
   private static final String SCHEMA_REGISTRY_SCOPE = SessionWindowsExampleTest.class.getName();
   private static final String MOCK_SCHEMA_REGISTRY_URL = "mock://" + SCHEMA_REGISTRY_SCOPE;
 
-  private TopologyTestDriver streams;
+  private TopologyTestDriver topologyTestDriver;
   private final Map<String, String> AVRO_SERDE_CONFIG = Collections.singletonMap(
     AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, MOCK_SCHEMA_REGISTRY_URL
   );
 
   @Before
   public void createStreams() {
-    streams = new TopologyTestDriver(
+    topologyTestDriver = new TopologyTestDriver(
       SessionWindowsExample.buildTopology(AVRO_SERDE_CONFIG),
       SessionWindowsExample.streamsConfig("dummy", TestUtils.tempDirectory().getPath())
     );
   }
 
   @After
   public void closeStreams() {
-    streams.close();
+    topologyTestDriver.close();
     MockSchemaRegistry.dropScope(SCHEMA_REGISTRY_SCOPE);
   }
 
   @Test
-  public void shouldCountPlayEventsBySession() throws Exception {
+  public void shouldCountPlayEventsBySession() {
     final SpecificAvroSerializer<PlayEvent> playEventSerializer = new SpecificAvroSerializer<>();
     playEventSerializer.configure(AVRO_SERDE_CONFIG, false);
 
     final long start = System.currentTimeMillis();
 
     final String userId = "erica";
-    IntegrationTestUtils.produceKeyValuesSynchronously(
-      SessionWindowsExample.PLAY_EVENTS,
-      Collections.singletonList(
-        new KeyValue<>(userId, new PlayEvent(1L, 10L))
-      ),
-      streams,
-      new StringSerializer(),
-      playEventSerializer,
-      start
-    );
+    topologyTestDriver.createInputTopic(SessionWindowsExample.PLAY_EVENTS,
```

Comment:
maybe we can also create the input and output topics in the `@Before` method so we can tidy up these tests.